test/unit/org/apache/cassandra/index/sai/cql/BaseDataModel.java

```diff
@@ -169,14 +169,22 @@ public void createTables(Executor tester)
 
     public void createIndexes(Executor tester) throws Throwable
     {
-        String template = "CREATE CUSTOM INDEX ndi_%s_index_%s ON %%s (%s) USING 'StorageAttachedIndex'";
+        createIndexes(tester, columns);
+    }
+
+    protected void createIndexes(Executor tester, List<Pair<String, String>> columns) throws Throwable
+    {
+        String indexNameTemplate = "ndi_%s_index_%s";
```

Comment:
nit: Ha, missed this earlier. "ndi" => "sai" I guess

Reply:
Fixed, this one and a similar one in `OperationTest.STATIC_CF_NAME`.